kingpin/actors/aws/ecs.py

```diff
@@ -422,31 +798,126 @@ def _get_containers_from_tasks(task_list):
                 }
             ]
 
-    Returns: All containers as a flat list.
-
+    Returns:
+        All containers as a flat list.
     """
     containers = []
     for task_overview in task_list:
         containers.extend(task_overview['containers'])
     return containers
 
 
-def _load_task_definition(task_definition_file, parameters):
+def _load_task_definition(task_definition_file, tokens):
     """Loads and verifies a task definition template file, and interpolates
-    parameters.
+    tokens.
 
     Args:
         task_definition_file: task definition file to load.
-        parameters: dict of key/value pairs to interpolate into the file.
+        tokens: dict of key/value pairs to interpolate into the file.
 
     Returns:
-        Parsed and interpolated task definition dict.
+        Resulting task definition dict.
     """
-    task_definition_file = utils.convert_script_to_dict(
-        task_definition_file, parameters)
+    task_definition = utils.convert_script_to_dict(
+        task_definition_file, tokens)
     try:
-        jsonschema.validate(task_definition_file,
+        jsonschema.validate(task_definition,
                             TASK_DEFINITION_SCHEMA)
     except jsonschema.exceptions.ValidationError as e:
         raise exceptions.InvalidOptions(e)
-    return task_definition_file
+    return task_definition
+
+
+def _load_service_definition(service_definition_file, tokens):
```

Comment:
I'm not a fan of these utility functions not being inside one of the classes. Is there a good reason they aren't either a) inside a utility package, or b) inside the ECSBaseActor class?